plugins/outputs/cloudwatch/cloudwatch_test.go

```diff
@@ -531,54 +539,38 @@ func TestBackoffRetries(t *testing.T) {
 	for i := 0; i <= defaultRetryCount; i++ {
```

Comment:
Maybe add `assert := assert.New(t)` to avoid using many `t` as recommended by [the package itself](https://pkg.go.dev/github.com/stretchr/testify/assert#hdr-Example_Usage)

Reply:
will fix